client/app/lib/onboarding.coffee

```diff
@@ -0,0 +1,111 @@
+# Local class Step
+class Step
+    # Retrieves properties from config Step plain object
+    constructor: (step) ->
+        if step
```

Comment:
It's more Vanilla JS economy habit, Coffee will be transpiled with more lines of code to implements this feature. But ok let's use it.